docs/adfs/README.md

```diff
@@ -51,12 +51,12 @@ passport.use(
       authnContext: [
         "http://schemas.microsoft.com/ws/2008/06/identity/authenticationmethod/password",
       ],
-      // not sure if this is necessary?
-      acceptedClockSkewMs: -1,
```

Comment:
[node-saml/node-saml/docs/adfs @ <= v4.0.0-beta.3](https://github.com/node-saml/node-saml/tree/d8bf6b55a70044d98eb767123bcf717808d7369d/docs/adfs#create-a-passport-framework) has this same insecure example (that same `node-saml/node-saml` document contains string as value for `authnContext`).